bindings/c/fdb_c.cpp

```diff
@@ -625,6 +625,19 @@ extern "C" DLLEXPORT FDBFuture* fdb_database_blobbify_range(FDBDatabase* db,
 	                        .extractPtr());
```

Comment:
My understanding (which admittedly could be wrong) was that if a client with the new release c or java api wanted to talk to an old release cluster, the old external client only has the original function, so the multi version client needs to call DLTransaction::blobbifyRange without the v2.

Reply:
I thought it was not released yet, because you add the comment that the other one is obsolete. If it is released already then better keep it. In that case instead of introducing the wait parameter, just introduce an API version that waits, e.g. fdb_database_blobbify_blocking

Reply:
Ok, I can switch it to a new call instead of a flag

Reply:
This is done

Reply:
Thanks